ruby_on_rails/README.md

```diff
@@ -32,7 +32,10 @@ They are not optional, but the rest of the team can start working even if those
 
 1. [GetSentry](getsentry.md)
 1. [NewRelic](newrelic.md)
-1. [Uptime Monitor](uptime.md)
+
+Some services should be enabled only if the customer pays for the monitoring package:
+
+1. [Uptime](uptime.md)
```

Comment:
can we link here the document with the monitoring overview? https://docs.google.com/spreadsheets/d/1FY4jqByO-aI5sDan0hD7ULu6a2-eLsmO6kgdCFlPmuY/edit#gid=0

if there is uptime SMS, then the "high priority" contact should be chosen

also based on the document we should enable depfu and set the sentry monitoring team